pkg/oc/bootstrap/docker/up.go

```diff
@@ -797,34 +803,22 @@ func (c *CommonStartConfig) CheckDockerInsecureRegistry(out io.Writer) error {
 
 // CheckNsenterMounter checks whether the Docker host can use the nsenter mounter from Kubernetes.
 // Otherwise, a shared volume is needed in Docker
-func (c *CommonStartConfig) CheckNsenterMounter(out io.Writer) error {
-	useNsenterMount, err := c.HostHelper().CanUseNsenterMounter()
-	if err != nil {
-		return err
-	}
-	if useNsenterMount && c.isRHDocker {
-		c.UseNsenterMount = true
-		fmt.Fprintf(out, "Using nsenter mounter for OpenShift volumes\n")
-	} else {
-		c.UseNsenterMount = false
-		fmt.Fprintf(out, "Using Docker shared volumes for OpenShift volumes\n")
-	}
-	return nil
+func (c *CommonStartConfig) CheckNsenterMounter(out io.Writer) (bool, error) {
```

Comment:
Doesn't require `out` and can probably be private.  Also, can't we just call the `c.HostHelper().CanUseNsenterMounter()` now?